astropy/coordinates/attributes.py

```diff
@@ -80,7 +79,7 @@ def convert_input(self, value):
 
         Returns
         -------
-        output_value
+        output_value : type
```

Comment:
I'm changing this to "output_value : object", which is the least-informative, but always true, return type.
I felt I had to put something there because the NumPyDocs style guide — to which X-ref adheres — says all Return values must specify the type.

<img width="664" alt="Screen Shot 2021-03-30 at 12 49 54" src="https://user-images.githubusercontent.com/8949649/113025873-6e610880-9156-11eb-980c-3647aaa2d1dd.png">